grama/tools.py

```diff
@@ -308,9 +308,10 @@ def add_pipe(fun):
     class NewPipe(pipe):
         __name__ = fun.__name__
         __doc__ = (
-            "Pipe-enabled version of {}\n".format(fun.__name__)
-            + "Inherited Signature: {}\n".format(signature(fun))
-            + fun.__doc__
+            fun.__doc__
```

Comment:
This creates a new issue that we'll have to resolve....

Reply:
@OscarDeGar can you open a new Issue to track this? Last thing to fix before moving on....